projects/jetty/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Google LLC
+# Copyright 2023 Google LLC
```

Comment:
We don't change the year when editing the file

Reply:
@jonathanmetzman undone.